examples/src/test/resources/projects/librarymanagement.conf

```diff
@@ -10,3 +10,5 @@ modules {
 }
 
 runner = "ScalaTests"
+
+probe.paths.screenshots = ${?IDEPROBE_SCREENSHOTS_DIR}
```

Comment:
this should be in probe block in line 4-5, but actually is it needed at all if we include ideprobe.conf?

Reply:
it's not necessary there